Sources/Recycling/GridViews/Ads/Cell/AdsGridViewCell.swift

```diff
@@ -263,6 +271,13 @@ public class AdsGridViewCell: UICollectionViewCell {
         let colors: [UIColor] = [.toothPaste, .mint, .banana, .salmon]
         let color = colors[index % 4]
         loadingColor = color
+
+        if let model = model {
+            if !model.canScaleImageToFillView {
```

Comment:
I would prefer to have this test for `true` instead.
`if model.scaleImageToFitView {`

Reply:
Thanks for the feedback! I agree with the two above. However, in this one, the model does not have a scaleImageToFitView variable. Since the default is to have scaleImageToFill, I think it makes sense to keep that one instead of changing the variable to scaleImageToFitView. Right now, ios-app sets scaleImageToFillView to true for all markets except jobs.